Checks/PWR003/gravity.c

```diff
@@ -0,0 +1,4 @@
+// Returns the gravity on Earth in m/s2
+double gravity() {
+  return 9.8;
+}
```

Comment:
I feel like this function is not very realistic. In particular, I doubt if a programmer would first think of enclosing a single value in a function, rather than directly declaring it as a constant in the code.

To give it a twist and make it more realistic, I think we could make the function able to return different gravities; for example, for all the planets in the solar system. Thus, it would also receive an argument that would control the return value. No need to do anything fancy, we can receive an `int` to keep it simple, and act with a `switch`/`select case` construct.

As long as the benchmark always requests gravity for the same planet, it should still get the benefits of a pure function.

WDYT?

Reply:
Okay, i'll make this change, as provides realism, I'll add an `int` for simplicity but I'll leave a comment just to clarify